recipes/the-silver-searcher/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "the-silver-searcher" %}
+{% set version = "2.0.0" %}
+{% set sha256 = "ff7243863f22ed73eeab6f7a6d17cfff585a7eaa41d5ab3ae4f5d6db97701d5f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/ggreer/{{ name | replace('-', '_') }}/archive/{{ version }}.tar.gz
```

Comment:
GitHub archive files are dynamically generated every time a user downloads them. So there is no guarantee the checksum will remain the same. GitHub has told us before that they try to keep the checksum unchanged, but will not guarantee the checksum will not change. We have also seen cases where the checksum changes even though nothing has happened to the tag in question. These are normally benign things like timestamps changing or file ordering changing. That said, it is still concerning that it happens at all.